### PR TITLE
ADH Branding

### PR DESCRIPTION
**Pull request by anton-dahlin**

### CODE REVIEW

**Review by derekendres — APPROVED** [review]

File: src/app/models/sds-namespace.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /**
  * SDS Namespace representation, see
- * {@link https://ocs-docs.osisoft.com/Content_Portal/Documentation/Management/Account_Namespace_1.html|OCS Documentation},
+ * {@link https://ocs-docs.osisoft.com/Content_Portal/Documentation/Management/Account_Namespace_1.html|ADH Documentation},
```
Comment (kelseyduffy) [inline src/app/models/sds-namespace.ts:3]: it looks like all of these links are not great to use anymore. If you go to any of them, it renders the OCS Content Portal for a few seconds, then it auto routes you to the OCS Documention main landing page. These are therefore effectively dead links.

Comment (kelseyduffy) [inline src/app/models/sds-namespace.ts:3]: separately, I don't really like the idea of saying "ADH Documentation" when said documentation doesn't mention ADH anywhere. In a literal sense, this is still "OCS" documentation. Rebranding this before the docs are rebranded seems a bit awkward

Comment (anton-dahlin) [reply src/app/models/sds-namespace.ts:3]: Turns out they just launched the ADH portion of the docs so with the new links this should be good

**Review by kelseyduffy — APPROVED** [review]

**Pull request merged**